examples/openshift-origin/README.md

```diff
@@ -0,0 +1,64 @@
+## OpenShift Origin example
+
+This example shows how to run OpenShift Origin as a pod on an existing Kubernetes cluster.
+
+This example demonstrates usage of a pod with a secret volume mount.
+
+### Step 0: Prerequisites
+
+This example assumes that you have a basic understanding of Kubernetes and that you have forked the repository and [turned up a Kubernetes cluster](https://github.com/GoogleCloudPlatform/kubernetes#contents):
+
+This example has been tested against the **gce** and **vagrant** based KUBERNETES_PROVIDER.
+
+```shell
+$ cd kubernetes
+$ export KUBERNETES_PROVIDER=gce
+$ hack/dev-build-and-up.sh
+```
+
+### Step 1: Generate secrets
+
+The OpenShift Origin pod requires the ability to securely authenticate with the underlying Kubernetes provider.
+
+We will use Kubernetes Secrets to support providing this information to the OpenShift Origin pod.
+
+To generate a secret file that can contain the information needed to authenticate to your Kubernetes provider, run the following:
+
+```shell
+$ examples/openshift-origin/secret-generator.sh
+```
+
+This will generate a secret resource that will contain all the information needed by a .kubeconfig file in order to authenticate with a provider.
```

Comment:
> generate a new .kubeconfig file that contains a targeted set of credentials

Can we get some examples of how the credential requestor and the generated credential identity is specified?  Does the user we're generating for have to exist ahead of  time?  Do we need to bind roles to the identity?